tests/test_s3.py

```diff
@@ -129,7 +130,9 @@ def test_instantiation(self):
 
     def test_empty_string(self):
         # Arrange
-        settings = {'username': '', 'password': ''}
+        username = 'username'
+        password = 'password'
+        settings = {'username': username, 'password': password}
```

Comment:
What is the `empty_string` here if not the credentials? Should this be called something like `test_unsigned_uploader`?

Reply:
Renamed the test method and added additional cases